clusterbuster

```diff
@@ -710,6 +717,12 @@ function process_option() {
 		*) affinity=0         ;;
 	    esac
 	    ;;
+        # vm specs
+        vmcores)                    vm_cores=$optvalue                          ;;
+        vmsockets)                  vm_sockets=$optvalue                        ;;
+        vmthreads)                  vm_threads=$optvalue                        ;;
+        vmgraceperiod)              vm_grace_period=$optvalue                   ;;
+        vmimage)                    vm_image=$optvalue                          ;;
```

Comment:
These options need to be documented in _help_extended.  Make certain that when you add them there that you use only spaces for indentation, not tab characters.

Reply:
added the help messages under Virtualization Options.